### PR TITLE
[Validator] Add missing Serbian translation

### LINKED ISSUE #53043: Missing translations for Serbian (sr_Latn)
Hello,

There are some missing Serbian (sr_Latn) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Serbian (sr_Latn) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by marac19901990**
[…ions](https://github.com/symfony/symfony/issues/53043)

| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #53043 
| License       | MIT

Added one new missing translation, and reformulated the wording on old ones. This is my native language so i wanted to make it as clear as possible.

### CODE REVIEW

**Comment by Nyholm** [comment]
@ajakov can you please help review this?

**Comment by ajakov** [comment]
Yes, I will look into it.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 1]
```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Vrednost bi trebalo da bude netačna.</target>
+                <target>Vrednost bi trebala da bude netačna.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:7]: You are right, this is the correct form for this verb when subject is preceding it. Could you change every occurence of it, where applicable?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 2]
```diff
@@ -20,19 +20,19 @@
             </trans-unit>
             <trans-unit id="5">
                 <source>The value you selected is not a valid choice.</source>
-                <target>Odabrana vrednost nije validan izbor.</target>
+                <target>Vrednost koju ste izabrali nije validan izbor.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:23]: It's the same, but this sounds better. 👍

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 3]
```diff
@@ -20,19 +20,19 @@
...
             </trans-unit>
             <trans-unit id="7">
                 <source>You must select at most {{ limit }} choice.|You must select at most {{ limit }} choices.</source>
-                <target>Morate odabrati najviše {{ limit }} mogućnost.|Morate odabrati najviše {{ limit }} mogućnosti.|Morate odabrati najviše {{ limit }} mogućnosti.</target>
+                <target>Morate izabrati najviše {{ limit }} mogućnosti.|Morate izabrati najviše {{ limit }} mogućnosti.</target>
             </trans-unit>
             <trans-unit id="8">
                 <source>One or more of the given values is invalid.</source>
-                <target>Jedna ili više vrednosti nisu validne.</target>
+                <target>Jedna ili više ponuđenih vrednosti nisu validne.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Now, I am not sure about this. What is the real meaning of this? Is it `ponuđenih` or `odabranih`. Maybe we could use `datih`?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 4]
```diff
@@ -44,59 +44,59 @@
...
+                <target>Vrednost nije validna kao datum i vreme.</target>
             </trans-unit>
             <trans-unit id="13">
                 <source>This value is not a valid email address.</source>
                 <target>Vrednost nije validna adresa elektronske pošte.</target>
             </trans-unit>
             <trans-unit id="14">
                 <source>The file could not be found.</source>
-                <target>Datoteka ne može biti pronađena.</target>
+                <target>Fajl ne može biti pronađen.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:59]: OK, `fajl` is definitely more recognisable to the broad audience than `datotetka`. 👍

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 5]
```diff
@@ -44,59 +44,59 @@
...
+                <target>Vrednost je prekratka. Trebalo bi da ima {{ limit }} karaktera ili više.|Vrednost je prekratka. Trebalo bi da ima {{ limit }} karaktera ili više.</target>
             </trans-unit>
             <trans-unit id="22">
                 <source>This value should not be blank.</source>
                 <target>Vrednost ne bi trebalo da bude prazna.</target>
             </trans-unit>
             <trans-unit id="23">
                 <source>This value should not be null.</source>
-                <target>Vrednost ne bi trebalo da bude prazna.</target>
+                <target>Vrednost ne bi trebalo da bude null.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Should this also be `trebala`?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 6]
```diff
@@ -172,79 +172,79 @@
             </trans-unit>
             <trans-unit id="46">
                 <source>The image height is too small ({{ height }}px). Minimum height expected is {{ min_height }}px.</source>
-                <target>Visina slike je premala ({{ height }} piksela). Najmanja dozvoljena visina je {{ min_height }} piksela.</target>
+                <target>Visina slike je preniska ({{ height }} piksela). Najmanja dozvoljena visina je {{ min_height }} piksela.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:175]: 👍

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 7]
```diff
@@ -172,79 +172,79 @@
...
             </trans-unit>
             <trans-unit id="61">
                 <source>This value is not a valid ISBN-13.</source>
-                <target>Ovo nije validan ISBN-13.</target>
+                <target>Nevalidna vrednost ISBN-13.</target>
             </trans-unit>
             <trans-unit id="62">
                 <source>This value is neither a valid ISBN-10 nor a valid ISBN-13.</source>
-                <target>Ovo nije validan ISBN-10 ili ISBN-13.</target>
+                <target>Vrednost nije ni validan ISBN-10 ni validan ISBN-13.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:239]: 👍

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 8]
```diff
@@ -372,39 +372,39 @@
...
+                <target>Broj elemenata u ovoj kolekciji bi trebalo da bude višestruk u odnosu na {{ compared_value }}.</target>
             </trans-unit>
             <trans-unit id="97">
                 <source>This value should satisfy at least one of the following constraints:</source>
                 <target>Ova vrednost bi trebalo da zadovoljava namjanje jedno od narednih ograničenja:</target>
             </trans-unit>
             <trans-unit id="98">
                 <source>Each element of this collection should satisfy its own set of constraints.</source>
-                <target>Svaki element ove kolekcije bi trebalo da zadovolji sopstveni skup ograničenja.</target>
+                <target>Svaki element ove kolekcije bi trebalo da zadovolji sopstveni set ograničenja.</target>
```
Comment (ajakov) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: I think `skup` is a well-known translation for `set`. We should keep `skup` here.

**Review by ajakov — COMMENTED** [review]
Hey @marac19901990 I have some comments. With some small modifications, I would accept this. 
Overall, great work, thank you!

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 1 above)
Comment (marac19901990) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:7]: @ajakov sure commit coming :)

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 8 above)
Comment (marac19901990) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: @ajakov You had great suggestions, thank you :)

**Review by ajakov — APPROVED** [review]
Cool, we can go on with this one.

**Comment by nicolas-grekas** [comment]
Thank you @marac19901990.

**Pull request merged**